recipes-kernel/linux/files/acrn-common.cfg

```diff
@@ -1 +1,3 @@
 CONFIG_ACRN_VIRTIO_DEVICES=y
+CONFIG_UIO=m
+CONFIG_UIO_PCI_GENERIC=m
```

Comment:
Are these configurations applicable to both SOS and UOS ?
Are these configurations applicable to both 4.19 and 5.4 kernels ?

Reply:
Only for UOS, for both of 4.19 and 5.4 kernels

Reply:
@saininav  please give some advice about this PR, which is just for UOS. I found that sos also use acrn-common.cfg, so this PR  need modify. Could you please help generate new PR for this, both on master-next and dunfell ? just close 122 and 123, thanks